models/commit_status.go

```diff
@@ -54,14 +54,14 @@ const (
 // CommitStatus holds a single Status of a single Commit
 type CommitStatus struct {
 	ID          int64             `xorm:"pk autoincr"`
-	Index       int64             `xorm:"INDEX UNIQUE(repo_sha_index)"`
-	RepoID      int64             `xorm:"INDEX UNIQUE(repo_sha_index)"`
+	RepoID      int64             `xorm:"UNIQUE(repo_sha_index)"`
```

Comment:
Why remove this `index`?

Reply:
I've reordered the struct in order to need only one index.

Before:

* **UQE_commit_status_repo_sha_index** ==> `index`, `repo_id`, `sha`; only useful for ensuring uniqueness. Not really for queries as `index` is not very unique.
* **IDX_commit_status_repo_id** ==> needed for most queries, because they all use `WHERE repo_id = x` as condition (but see my note below).

After:

* **UQE_commit_status_repo_sha_index** ==> `repo_id`, `sha`, `index`; ensures uniqueness _and_ can be used for queries like `WHERE repo_id = x`.

Note: now that I think of it, `UQE_commit_status_repo_sha_index` should be `sha, index, repo_id` because `sha` is always part of the query condition and it's more specific than `repo_id`. I'll change that.

Reply:
In my latest commit `UQE_commit_status_repo_sha_index` is now `sha, index, repo_id`.